src/InputFormat.jsx

```diff
@@ -1,10 +1,10 @@
 import { FormControl, InputLabel, Input } from "@material-ui/core";
-
+//input 부분 하나로 묶음
 export default function InputFormat(props){
```

Comment:
이렇게 합시다.

```suggestion
/* props: {
     value: string;
     onChange: (event) => void;
   }
 */
export default function InputFormat(props){
```

지금 아직 우리 코딩 스타일을 정한 게 없어서 이렇게 표시하면 좋은것들은 하나씩 추가해나갑시다

Reply:
확인했습니다